defold-rive/src/comp_rive.cpp

```diff
@@ -426,23 +426,33 @@ namespace dmRive
                     ro.m_SetBlendFactors = 1;
 
                     const rive::PaintData draw_entry_paint = rive::getPaintData(ctx->m_Paint);
-                    const float* color                     = &draw_entry_paint.m_Colors[0];
 
                     dmVMath::Vector4 properties((float)draw_entry_paint.m_FillType, (float)draw_entry_paint.m_StopCount, 0.0f, 0.0f);
                     dmVMath::Matrix4 local_matrix;
                     Mat2DToMat4(ctx->m_Event.m_TransformLocal, local_matrix);
 
+                    dmVMath::Vector4 colors[rive::PaintData::MAX_STOPS];
```

Comment:
We cannot simply cast from a "float*" to a Vector4* as the alignment of the struct is different!